pkg/migrations/mls/20231023050806_init-schema.up.sql

```diff
@@ -0,0 +1,44 @@
+SET
+    statement_timeout = 0;
+
+--bun:split
+CREATE TABLE installations (
+    id TEXT PRIMARY KEY,
+    wallet_address TEXT NOT NULL,
+    created_at BIGINT NOT NULL,
+    revoked_at BIGINT
+);
+
+--bun:split
+CREATE TABLE key_packages (
+    id TEXT PRIMARY KEY,
+    installation_id TEXT NOT NULL,
+    created_at BIGINT NOT NULL,
+    consumed_at BIGINT,
+    is_last_resort BOOLEAN NOT NULL,
+    data BYTEA NOT NULL,
+    -- Add a foreign key constraint to ensure key packages cannot be added for unregistered installations
+    CONSTRAINT fk_installation_id FOREIGN KEY (installation_id) REFERENCES installations (id)
+);
+
+--bun:split
+CREATE INDEX idx_installations_wallet_address ON installations(wallet_address);
+
+--bun:split
+CREATE INDEX idx_installations_created_at ON installations(created_at);
+
+--bun:split
+CREATE INDEX idx_installations_revoked_at ON installations(revoked_at);
+
+--bun:split
+-- Adding indexes for the key_packages table
+CREATE INDEX idx_key_packages_installation_id ON key_packages(installation_id);
```

Comment:
Most of these individual indexes seem unnecessary based on the queries in `store.go`. I'm only see the need for these 2:
- `(installation_id, not_consumed, is_last_resort, created_at)`
- `(is_last_resort, id)`

Reply:
There are more methods coming to the store in the next PR that should use some of them. But I do like the idea of adding the `not_consumed` field and a compound index for the most common/expensive query. I'll add that and remove a few of the related indexes.